librz/core/cmd_descs/cmd_analysis.yaml

```
@@ -170,6 +170,44 @@ commands:
           - RZ_OUTPUT_MODE_STANDARD
           - RZ_OUTPUT_MODE_RIZIN
         args: []
+  - name: ad
+    summary: Analyze data
+    subcommands:
+      - name: ad
+        summary: Analyze [count] data words with [depth]
```

```suggestion
        summary: Analyze <count> data words with <depth>
```

If you use `<arg>` RzCmd automatically color the string.